docs/core-concepts/app-deployment.md

```diff
@@ -1,17 +1,255 @@
+              }
+          } 
+      }
+    }
+    ```
+
+3. Open the `project.json` file. Remove the following line in in the `dependencies` section:
+
+    ```project.json
+    "type": "platform",
+    ```
+
+4. Create a `runtimes` section in your `project.json` file that defines the platforms your app targets and specify the runtime identifier of each platform that you target. See [Runtime IDentifier catalog](https://dotnet.github.io/docs/core-concepts/rid-catalog.html "Runtime IDentifier (RID) catalog") for a list of runtime identifiers. For example, the following `runtimes` section indicates that the app runs on 64-bit Windows 10 operating systems and the 64-bit OSX Version 10.10 operating system.
+
+    ```project.json
+        "runtimes": {
+        "win10-x64": {},
+        "osx.10.10-x64": {}
+        }
+    ```
+Note that you also need to add a comma to separate the `runtimes` section from the previous section.
+A complete sample `project.json` file appears later in this section.
+
+5. Run the `dotnet restore` command to restore the dependencies specified in your project.
+
+6. Create debug builds of your app on each of the target platforms by using the `dotnet build -r` command and specifying the runtime directive of the platform. In our example, we would run two separate builds as follows:
+
+    ```project.json
+    dotnet build -r win10-x64
+    dotnet build -r osx.10.10-x64
+    ```
+
+7. After you've debugged and tested the program, you can create the files to be deployed with your app for each platform that it targets. To do this, use the `dotnet publish -c release -r` command. This creates a release (rather than a debug) version of your app for each target platform. In our example, we would publish our project for two separate platforms as follows:
+
+    ```project.json
+    dotnet publish -c release -r win10-x64
+    dotnet publish -c release -r osx.10.10-x64
+    ```
+   The resulting files are placed in a subdirectory named `publish` that is in a subdirectory of your project's `.\bin\release\netcoreapp1.0\<runtime_identifier>` subdirectory. Note that each subdirectory contains the complete set of files (both your app files and all .NET Core files) needed to launch your app.
+
+8. Along with your application's files, the publishing process emits a program database (.pdb) file that contains debugging information about your app. You should not package it with your application's files.
+
+The remaining files can be deployed in any way you'd like. For example, you can package them in a zip file, use a simple `copy` command, or deploy them with any installation package of your choice.
+
+The following is the complete `project.json` file for this project.
+
+```project.json
+    {
+      "version": "1.0.0-*",
+      "buildOptions": {
+        "emitEntryPoint": true
+      },
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "version": "1.0.0"

```

Comment:
Can get rid of object syntax here and just use `"Microsoft.NETCore.App": "1.0.0"`